common/game_rules/wc_unimportant_game_rules.txt

```diff
@@ -1,3 +1,17 @@
+patrons_game_rule = {
+	name = "RULE_PATRONS"
+	group = "RULE_GROUP_WARCRAFT"
+	option = {
+		name = exist
+		text = "RULE_PATRONS_EXIST"
+		desc = "RULE_PATRONS_EXIST_DESC"
+	}
+	option = {
+		name = patron_trait
+		text = "RULE_PATRONS_PATRON_TRAIT"
+		desc = "RULE_PATRONS_PATRON_TRAIT_DESC"
+	}
```

Comment:
Strange that `patron_trait` isn't default rule.

Reply:
Changed, but, imo, "by default" this trait feels like it's off-place and breaks immersion